package.json

```diff
@@ -164,6 +164,7 @@
 		"gray-matter": "^4.0.3",
 		"http-status-codes": "^2.2.0",
 		"isomorphic-unfetch": "^3.1.0",
+		"jest-raw-loader": "^1.0.1",
```

Comment:
@pbortnick can we move this to `devDependencies`? As it's only used during testing, it makes the most sense to shift it there!

Reply:
good call. Done! https://github.com/hashicorp/dev-portal/pull/1317/commits/f25dd062367b9994649fd50372dc5fd465c019c7